src/main/java/bisq/asset/coins/Mfcoin.java

```diff
@@ -0,0 +1,10 @@
+package bisq.asset.coins;
+
+import bisq.asset.Coin;
+import bisq.asset.RegexAddressValidator;
+
+public class Mfcoin extends Coin {
```

Comment:
Please name the class `MFCoin` if that is indeed the name of the project / coin, i.e. the way it is spelled on your website, etc.